src/server_manager/web_app/ui_components/app-root.js

```diff
@@ -36,17 +36,27 @@ import './outline-manual-server-entry.js';
 import './outline-modal-dialog.js';
 import './outline-region-picker-step.js';
 import './outline-server-progress-step.js';
-import './outline-server-view.js';
 import './outline-tos-view.js';
 
 import {AppLocalizeBehavior} from '@polymer/app-localize-behavior/app-localize-behavior.js';
 import {mixinBehaviors} from '@polymer/polymer/lib/legacy/class.js';
 import {html} from '@polymer/polymer/lib/utils/html-tag.js';
 import {PolymerElement} from '@polymer/polymer/polymer-element.js';
 
+import {ServerView} from './outline-server-view.js';
+
 const TOS_ACK_LOCAL_STORAGE_KEY = 'tos-ack';
 
-class AppRoot extends mixinBehaviors
+/**
+ * A server to be displayed
+ * @typedef {Object} DisplayServer
+ * @property {string} id - The id of the server
+ * @property {string} name - The display name of the server
+ * @property {boolean} isManaged - Whether the server host is managed by the Outline Manager
+ * @property {boolean=} isSynced - Whether the server information is updated
```

Comment:
Extra `=`?

Reply:
That's a JSDoc notation for "optional". But it turns out there's a clearer notation, to which I changed it now. I'm also using @prop, which is shorter.

Reply:
Got it!